system_up.sh

```diff
@@ -20,6 +20,8 @@ start_services() {
 		done
 	fi
 	echo "👷 Building service(s)"
+
+  cp ../Dockers/config/litefs.static-lease.yml ./config/
```

Comment:
since the torii docker image has a context of `contracts` , we copy it over to the contracts/config directory so it can be copied into the docker file